analysis/filter/StructureModifier/src/test/kotlin/de/maibornwolff/codecharta/filter/structuremodifier/NodeRemoverTest.kt

```diff
@@ -91,4 +94,22 @@ class NodeRemoverTest {
 
         Assertions.assertThat(result.blacklist.map { it.path }).doesNotContainSubsequence("/root/foo")
     }
+
+    @Test
+    fun `should copy attributeDescriptors`() {
```

Comment:
Small remark: It is not directly obvious for me what this method is testing. Suggestion: Add the reason why it should copy the descriptors.

Reply:
I will rename this test. This test verifies that the attribute descriptors do not get changed by a remove action, when there is no need to do so.